leap/leap_test.rb

```diff
@@ -15,26 +18,44 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert_equal true, Year.leap?(1996)
```

Comment:
If the value you are asserting against is `true` or `false`, it's idiomatic in minitest to use `assert` and `refute`.